### PR TITLE
Feat: Shared rich previews.

### PR DESCRIPTION
**Pull request by Jamerrone**

### CODE REVIEW

File: projects/browser-extension/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "dev": "run-p dev:* devtools",
     "dev:app": "webpack --watch",
     "dev:examples": "serve examples/",
-    "prebuild": "rimraf ./dist",
+    "prebuild": "rimraf ./dist && mkdir -p ./dist/previews && cp -R ../rich-previews/dist/** ./dist/previews",
```
Comment (bob-voorhoede) [inline projects/browser-extension/package.json:12]: This assumes that the previews are already built. We need to figure out the proper order to build the previews, as this needs to happen for every other app, and needs to happen before those apps start building/devving.

File: projects/rich-previews/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  'parserOptions': { 'ecmaVersion': 2017 },
+  'env': { 'es6': true },
```
Comment (bob-voorhoede) [inline projects/rich-previews/.eslintrc.js:3]: Why these?

File: projects/rich-previews/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  node: { fs: 'empty' },
```
Comment (bob-voorhoede) [inline projects/rich-previews/webpack.config.js]: What does this do?

**Review by bob-voorhoede — APPROVED** [review]
Cool! ✅ 
I have a few comments, mostly out of curiosity.
We need to figure out the correct build pipeline, so certain stuff is built and ready for other apps to use; but that can be done at a later stage.

File: projects/browser-extension/package.json (hunk 1 above)
Comment (Jamerrone) [reply projects/browser-extension/package.json:12]: Yeah... for now I have added a temporary build script in the root that builds everything in order.

File: projects/rich-previews/webpack.config.js (hunk 3 above)
Comment (Jamerrone) [reply projects/rich-previews/webpack.config.js]: Woops, this was copied from the other config, I am not really sure. I will try to remove it.

**Review by bob-voorhoede — APPROVED** [review]
✅

**Pull request merged**